HotelPriceScout/Pages/DashboardPage.razor

```diff
@@ -233,65 +260,66 @@
 
 @code
 {
+    private bool _dropdown;
+    decimal _marketPrice;
+    readonly List<PriceModel> _marketPriceList = new List<PriceModel>();
+    decimal _kompasPrice;
+    private int _adultsPicked = 1;
+
+    private IEnumerable<PriceModel> _monthData;
+    private IEnumerable<PriceModel> _monthMarketData;
+    private IEnumerable<PriceModel> _monthKompasPrices;
+    
     private void RemoveCard(WarningMessage popup)
     {
-        Dashboard.WarningMessages.Remove(popup);
+        _dashboard.WarningMessages.Remove(popup);
     }
```

Comment:
er den her metode nødvendig? Kan vi ikke bare udføre koden der hvor den kaldes?

Reply:
Ikke i det stadie, som den var i. Men nogen havde glemt at sørge for at pop ups blev vist, hvis de blev triggered. Vil du se commit e6f9d7cdff0d417f3aa8e98b0d62b5360ecdf739 igennem?

Reply:
ah i see, lgtm